src/sagas/wallet.js

```diff
@@ -531,10 +545,37 @@ export function* onWalletConnStateUpdate({ payload }) {
   yield put(isOnlineUpdate({ isOnline }));
 }
 
+export function* walletReloading() {
+  const wallet = yield select((state) => state.wallet);
+  const routerHistory = yield select((state) => state.routerHistory);
+
+  // Since we close the channel after a walletReady event is received,
```

Comment:
Are we closing this channel in this method after the reload is completed?

Reply:
No, the channel closes itself after the wallet is ready or erroed, so we don't have to